src/bp-core/bp-core-functions.php

```diff
@@ -4708,11 +4708,13 @@ function bp_core_parse_url( $url ) {
 		$parsed_url_data['wp_embed']    = true;
 	} else {
 
+		$user_agent = apply_filters( 'http_headers_useragent', 'WordPress/' . get_bloginfo( 'version' ) . '; ' . get_bloginfo( 'url' ), $url );
+
 		// safely get URL and response body.
 		$response = wp_safe_remote_get(
 			$url,
 			array(
-				'user-agent' => '', // Default value being blocked by Cloudflare
+				'user-agent' => $user_agent, // Default value being blocked by Cloudflare
```

Comment:
@mahedihasannoman We have intentionally made the user-agent as blank here.
Whatever code you had added it's default by WP itself. https://prnt.sc/K6aAUrkXpo91

Can you please explain what exactly you are trying to do here with the default user agent?

Reply:
@KartikSuthar Maybe but it does not work at all. You can try with any Shopify shop URL as an example like this one: https://nimb.ws/gwBz0f
But if you try with this PR the Shopify shop URL should accessible.

Reply:
@mahedihasannoman If this is the case then, it will also be going to work when we comment the 
```
// 'user-agent' => $user_agent, 
```
as which user agent you are passing it will be passed by the WordPress itself. Check the screenshot: https://prnt.sc/K6aAUrkXpo91